docs/getting-started/quickstart.md

```diff
@@ -35,100 +35,175 @@ export NVM_DIR="$HOME/.nvm"
 
+<Tabs groupId="postgres-or-docker">
+  <TabItem value="postgres" label="Install Postgres Locally">
+      Follow these [official guides](https://www.postgresql.org/download/linux/) to install Postgres on several popular Linux distributions.
   </TabItem>
-  <TabItem value="win-ps" label="Windows">
+    <TabItem value="docker" label="Launch Postgres with Docker">
+      Follow the [Docker Engine installation page](https://docs.docker.com/engine/install/) to install Docker on several popular Linux distributions.
+      Later in this guide, we'll provide instructions on launching Postgres with Docker.
+   </TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="win-ps" label="Windows">
+
+<h3>Installing Node.js</h3>
 
-   First, download and install [Node.js 20 or later](https://nodejs.org/en).
-   After installing Node.js, manually create the following folder required by `npm`: `C:\Users\%user%\AppData\Roaming\npm`
-   (`%user%` is the Windows user on which you are logged in).
+Download Node.js 20 or later from the [official Node.js download page](https://nodejs.org/en/download) and install it.
+After installing Node.js, create the following folder: `C:\Users\%user%\AppData\Roaming\npm`
+(`%user%` is the Windows user on which you are logged in).
 
-   Then, download and install [Docker Desktop for Windows](https://docs.docker.com/desktop/install/windows-install/).
+<h3> Installing Postgres </h3>
+
+You can install Postgres locally or launch it in a Docker container.
+
+<Tabs groupId="postgres-or-docker">
+  <TabItem value="postgres" label="Install Postgres Locally">
+      Follow [this official guide](https://www.postgresql.org/download/windows/) to install Postgres on Windows.
   </TabItem>
-   
+    <TabItem value="docker" label="Launch Postgres with Docker">
+      You can install Docker on Windows through [Docker Desktop](https://docs.docker.com/desktop/install/windows-install/).
+      Later in this guide, we'll provide instructions on launching Postgres with Docker.
+   </TabItem>
 </Tabs>
 
-After installing Docker, close and reopen your terminal to apply any changes. Then, verify Docker is working by running:
-
-   ```bash
-docker run hello-world
-   ```
+</TabItem>   
+</Tabs>
 
 ## Project Initialization
 
 To initialize a new DBOS application, run the following command:
    ```bash
-npx -y @dbos-inc/create -n <app-name>
+npx -y @dbos-inc/create@latest -n <app-name>
    ```
 
 Application names should be 3 to 30 characters long and contain only lowercase letters and numbers, dashes, and underscores.
 
 This creates a folder for your application, configures its layout, and installs required dependencies.
 If successful, it should print `Application initialized successfully!`.
-By default, `init` instantiates a "Hello, Database!" application which greets users and tracks the count of greetings per user.
+By default, it instantiates a "Hello, Database!" application which greets users and tracks the count of greetings per user.
```

Comment:
Maybe we can point people to two different next steps: 
- Running Locally
- Deploying to DBOS Cloud
This tutorial does seem to be longer than before..